core-it-suite/src/test/resources/mng-6957-buildconsumer/expected/simple-parent.pom

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?><!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+--><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+  	<groupId>org.sonatype.mavenbook.multi</groupId>
+  	<artifactId>parent</artifactId>
+  	<version>0.9-MNG6957-SNAPSHOT</version>
+  </parent>
+  <artifactId>simple-parent</artifactId>
+  <packaging>pom</packaging>
+  <name>Multi Chapter Simple Parent Project</name>
+ 
```

Comment:
Out of curiosity, the empty lines are expected/intended outcome?

Reply:
Good that you noticed this. You need to compare this with the `simple-parent/pom.xml`.
Because it is a stream, you'll always keep the whitespaces between `</name>` and `<modules>`. Keep in mind these aren't trimmed. 
This combined with the white spaces between `</modules>` and `<build>` makes it look like a large open block.
It might be possible to remove those last whitespace, if we can manage to see those last whitespaces as part of the modules-block. Not a real up-for-grabs, but a nice isolated Java challenge.

Reply:
Oh well, it isn't much of an issue, just something that caught my attention. If you say it's expected, it's fine.